Content.Server/Weapons/Melee/WeaponRandom/WeaponRandomComponent.cs

```diff
@@ -0,0 +1,40 @@
+using Content.Shared.Damage;
+using Robust.Shared.Audio;
+
+namespace Content.Server.Weapons.Melee.WeaponRandom;
+
+[RegisterComponent]
+internal sealed class WeaponRandomComponent : Component
```

Comment:
Confirming in case I missed something: this didn't get used in the final YAML? It seems quite silly as a concept.

Reply:
It is used in the rubberhammer, n ss13 it had an extremely small chance to instagib someone and initially i didnt plan to add this feature but people seemed to want it in. its about 1 in a million chance currently but can be changed easily enough.